packages/manager/modules/pci/src/projects/project/kubernetes/kubernetes.constants.js

```diff
@@ -1,7 +1,15 @@
 export const VERSION_ENUM_KEY = "models['cloud.kube.VersionEnum'].enum";
 export const ANTI_AFFINITY_MAX_NODES = 5;
+export const SCALE_DEFAULT_VALUES = {
+  LOWEST_MIN_VALUE: 0,
+  LOWEST_VALUE: 3,
+  DESIRED_VALUE: 3,
+  HIGHEST_VALUE: 100,
+  HIGHEST_MAX_VALUE: 100,
```

Comment:
Can't be exposed by API ?

Reply:
No, unfortunately there aren't default values returned from API.